src/components/InstructionDisplay.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import SimplePieChart from './SimplePieChart';
+import useCPU from '../hooks/use-cpu';
+
+const InstructionDisplay = () => {
+  const cpu = useCPU();
+
+  if (!cpu) {
+    return <h1>Loading...</h1>;
```

Comment:
in general, we should only have 1 h1 element on the page. perhaps make this a p tag instead. eventually though i think we will make "Loading..." text into spinners

Reply:
done